.github/workflows/ci.yaml

```diff
@@ -296,7 +296,7 @@ jobs:
             tests/.venv/
           key: ${{ runner.os }}-poetry-${{ hashFiles('**/Cargo.lock') }}
       - name: setup pytest
-        run: just pytest-setup
+        run: just venv
```

Comment:
> I did this because I wanted to keep the virtualenvs separate for the python tests and the bindings.

I have an idea for this one. Ill open up a followup pr for this shortly.